src/attrib/parse.ts

```diff
@@ -87,6 +99,75 @@ export async function parseItemFromAttribFile(file: string, data: any, civ: civC
       icon,
     };
 
+    if (type === ITEM_TYPES.ABILITIES && file.startsWith("abilities")) {
+      const translationParams = ui_ext.help_text_formatter?.formatter_arguments?.map((x) => Object.values(x)[0] ?? x) ?? [];
+      const effectsFactory = abilityModifiers[baseId];
+      const effects = effectsFactory?.(translationParams) ?? [];
+
+      // if (effects.length == 0) {
+      //   const addEffect = (property: ModifyableProperty, value: number, effect: "change" | "multiply" = "change", type: "passive" | "ability" = "passive") =>
+      //     effects.push({ property, value, effect, type });
+
+      //   for (const { id, value } of data?.ability_bag?.float_properties ?? []) {
+      //     if (id === "health_max") addEffect("hitpoints", value);
+      //     else if (id === "melee_damage") addEffect("meleeAttack", value);
+      //     else if (id === "charge_damage") addEffect("meleeAttack", value, "change", "ability");
+      //     else if (id === "armor_fire") addEffect("fireArmor", value);
+      //     else if (id === "armor_melee") addEffect("meleeArmor", value);
+      //     else if (id === "armor_range") addEffect("rangedArmor", value);
+      //     else if (id === "attack_speed") addEffect("attackSpeed", value);
+      //     else if (id === "damage") {
+      //       addEffect("meleeAttack", value);
+      //       addEffect("rangedAttack", value);
+      //       addEffect("siegeAttack", value);
+      //       addEffect("fireAttack", value);
+      //     } else if (id === "multiplier") {
+      //       // ignore
+      //     } else console.log("Unknown float property", id, value, item.attribName, item.id);
+      //   }
+      // }
+      
+      const ability: Ability = {
+        ...item,
+        type: "ability",
+        displayClasses: "",
+        classes: [],
+        // abilityType: file.split("/")[1].split("_")[0],
+        // activation: data.ability_bag.activation,
+        activation: file.split("/")[1].split("_")[0],
+        range: data.ability_bag.range / 4,
```

Comment:
Is this the casting range / influence range? Maybe something like 'aura', 'auraRange', or an other would work better?